static/src/javascripts/projects/common/modules/experiments/tests/https-test.js

```diff
@@ -0,0 +1,36 @@
+define([
+    'common/utils/cookies'
+], function (
+    cookies
+) {
+    return function () {
+        this.id = 'https-test';
+        this.start = '2016-04-28';
+        this.expiry = '2016-05-26';
+        this.author = 'Justin Pinner';
+        this.description = 'Test how users get on with https site-wide (0% initially)';
+        this.audience = 0;
+        this.audienceOffset = 0;
+        this.successMeasure = 'Pages are served over https';
+        this.audienceCriteria = 'Small percentage of users';
+        this.dataLinkNames = '';
+        this.idealOutcome = 'Nobody reports that things are broken';
+        this.hypothesis = 'We believe that most of our content should serve successfully over https, with a few known exceptions.';
+
+        this.canRun = function () {
+            return !cookies.get('https_opt_in');
```

Comment:
It's a pre-existing cookie which may or may not exist when the test runs. The original plan was to only execute this test when somebody was going to be moved into https. Now I think we do need a proper control group with the cookie set to false as long as it isn't already present. I just tested that and it doesn't appear to have any effect on existing https sections. Which is good.